langsecure/shield.py

```diff
@@ -32,6 +33,8 @@ def __init__(self, **params):
         else:
             # Load the policies into the pydantic class.
             self._py_policystore = store.PyPolicyStore(self.policy_store)
+        if self.tracking_server:
```

Comment:
can we always enable tracer which is defaulted to a file which could be in current dir or in ~/.langsecure/traces.log

If user passes the path then we use it or if langfuse server address is passed we use it.